### ISSUE TITLE
Timeline view start/end date

### ISSUE THREAD

**Issue by Xaero** [ActivityWatch/aw-webui#199]
I've used Manictime on Windows previously and have now switched to ActivtyWatch. The feature I'm currently missing that Manictime has is switching the activity view to a view like where you have the title start-end date and duration. 

![image](https://user-images.githubusercontent.com/64278556/83059473-3c74a680-a05a-11ea-9e3f-a309cf7fa1b9.png)

**Comment by ErikBjare** [comment]
Yeah this is something we're interested in fixing. It'll probably be part of the next web UI makeover, since there are some other related changes we'd like to make as well. 

I created an issue here with this and some other improvements we'd like to make: https://github.com/ActivityWatch/aw-webui/issues/200
